src/client/components/SearchFilterComponent/SearchFilter.styles.css

```diff
@@ -0,0 +1,210 @@
+  padding-left: 2.5rem;
+  position: relative;
+  right: 2.1rem;
+  border-style: solid;
+  border-width: 0.1rem;
+  border-color: #c4c4c4;
+}
+
+.search-filter-container h1 #a {
+  position: relative;
+  left: 14rem;
+  width: 10px;
+}
+.search-filter-container h1 #b {
+  position: relative;
+  left: 10rem;
+  width: 10px;
+}
+.search-filter-container h1 #c {
+  position: relative;
+  left: 5.5rem;
+  width: 10px;
+}
+
+hr {
+  position: relative;
+  margin-top: 2rem;
+  right: 0.9rem;
+  width: 22rem;
+  color: #c4c4c4;
+}
+.search-filter-container .input {
+  width: 320px;
+  height: 103px;
+}
+.search-filter-container .input input {
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+}
+.search-filter-container .input b {
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  margin-top: 0.7rem;
+  margin-left: 0.2rem;
+}
+
+/* minimum evaluation */
+
+.search-filter-container .minimum {
+  width: 320px;
+  height: 53px;
+  margin-top: 0rem;
+}
+
+.search-filter-container .inputMinimum {
+  width: 320px;
+  height: 163px;
+}
+
+.search-filter-container .inputMinimum input {
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+}
+
+.search-filter-container .inputMinimum b {
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  margin-top: 0.7rem;
+  margin-left: 0.2rem;
+}
+
+.search-filter-container .inputMinimum b img {
+  position: relative;
+  bottom: 0.1rem;
+  width: 10px;
+  left: 0.2rem;
+}
+/* number of reviews */
+
+.search-filter-container .review {
+  width: 320px;
+  height: 53px;
+  margin-top: 1.5rem;
+}
+
+.search-filter-container .review h1 {
+  border-color: #fdfbfb;
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+  font-size: 20px;
+  padding-top: 0.5rem;
+  padding-left: 2.5rem;
+  position: relative;
+  right: 2.1rem;
+}
+
+.slidecontainer {
+  width: 350px;
+  position: relative;
+  right: 0.5rem;
+}
+
+.slidecontainer #volume {
+  height: 15px;
+  width: 350px;
+  position: relative;
+  left: -0.1rem;
+}
+
+.range {
+  width: 350px;
+  display: flex;
+  margin-top: 0.5rem;
+  position: relative;
+  right: 0.5rem;
+}
+
+.range .cero {
+  position: relative;
+  left: 0rem;
+  padding-left: 0rem;
+  font-size: 18px;
+}
+
+.range .hundred {
+  position: relative;
+  left: 2.7rem;
+  padding-left: 15.7rem;
+  font-size: 18px;
+}
+
+@media screen and (max-width: 1200px) {
+  .search-filter-container {
+    display: flex;
+    flex-direction: column;
+    width: 387px;
+    padding: 1rem;
+    height: 1002px;
+    border-radius: 1.2rem;
+  }
+
+  .search-filter-container h1 {
+    margin-top: 0.7rem;
+    margin-left: 1.5rem;
+    font-size: 20px;
+    line-height: 21px;
+    width: 340px;
+    height: 40px;
+    padding-top: 0.5rem;
+    padding-left: 2.5rem;
+    position: relative;
+    right: 2.1rem;
+    border-style: solid;
+    border-width: 0.1rem;
+    border-color: #c4c4c4;
+  }
+}
+
+@media screen and (max-width: 750px) {
+  .search-filter-container {
+    background-color: #ffffff;
```

Comment:
Probably the background color doesn't change, so just have it outside of the media query.